Documentation/criu.txt

```diff
@@ -237,6 +237,12 @@ In other words, do not use it unless really needed.
     Tell *criu* that one end of a pair of UNIX sockets (created by
     *socketpair*(2)) with the given _id_ is OK to be disconnected.
 
+*--external* **pid[**__inode__**]:**__name__::
```

Comment:
maybe `pidns[inode]:name`?
I don't insist though.

Reply:
The reason for `pid` and not `pidns` is again the external network namespace support which uses `net`. I do not really care what name it has. Just following existing naming.